.circleci/config.yml

```diff
@@ -20,6 +20,9 @@ jobs:
       # - run:
       #     name: Run lint
       #     command: yarn lint
+      - run:
+           name: Build
+           command: yarn build
```

Comment:
It was all good while releasing without this build step, and could you please give more context on having this step?